<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SHA1CacheFinder</class>
 <widget class="QWidget" name="SHA1CacheFinder">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>505</width>
    <height>290</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SHA1 Cache Finder</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBoxProfile">
     <property name="title">
      <string>Profile</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_5">
      <item row="0" column="0">
       <widget class="QLabel" name="labelProfile">
        <property name="text">
         <string>Profile</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="comboBoxProfiles"/>
      </item>
      <item row="1" column="1">
       <widget class="QPushButton" name="pushButtonProfileManager">
        <property name="text">
         <string>Manager</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="labelProfileTID">
        <property name="text">
         <string>TID</string>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QLabel" name="labelProfileTIDValue"/>
      </item>
      <item row="1" column="2">
       <widget class="QLabel" name="labelProfileSID">
        <property name="text">
         <string>SID</string>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QLabel" name="labelProfileSIDValue"/>
      </item>
      <item row="0" column="4" rowspan="2">
       <widget class="Line" name="line_4">
        <property name="orientation">
         <enum>Qt::Orientation::Vertical</enum>
        </property>
       </widget>
      </item>
      <item row="0" column="5">
       <widget class="QLabel" name="labelProfileMACAddress">
        <property name="text">
         <string>MAC Address</string>
        </property>
       </widget>
      </item>
      <item row="0" column="6">
       <widget class="QLabel" name="labelProfileMACAddressValue"/>
      </item>
      <item row="1" column="5">
       <widget class="QLabel" name="labelProfileDSType">
        <property name="text">
         <string>DS Type</string>
        </property>
       </widget>
      </item>
      <item row="1" column="6">
       <widget class="QLabel" name="labelProfileDSTypeValue"/>
      </item>
      <item row="0" column="7" rowspan="2">
       <widget class="Line" name="line_3">
        <property name="orientation">
         <enum>Qt::Orientation::Vertical</enum>
        </property>
       </widget>
      </item>
      <item row="0" column="8">
       <widget class="QLabel" name="labelProfileVCount">
        <property name="text">
         <string>VCount</string>
        </property>
       </widget>
      </item>
      <item row="0" column="9">
       <widget class="QLabel" name="labelProfileVCountValue"/>
      </item>
      <item row="1" column="8">
       <widget class="QLabel" name="labelProfileTimer0">
        <property name="text">
         <string>Timer0</string>
        </property>
       </widget>
      </item>
      <item row="1" column="9">
       <widget class="QLabel" name="labelProfileTimer0Value"/>
      </item>
      <item row="0" column="10" rowspan="2">
       <widget class="Line" name="line_2">
        <property name="orientation">
         <enum>Qt::Orientation::Vertical</enum>
        </property>
       </widget>
      </item>
      <item row="0" column="11">
       <widget class="QLabel" name="labelProfileGxStat">
        <property name="text">
         <string>GxStat</string>
        </property>
       </widget>
      </item>
      <item row="0" column="12">
       <widget class="QLabel" name="labelProfileGxStatValue"/>
      </item>
      <item row="1" column="11">
       <widget class="QLabel" name="labelProfileVFrame">
        <property name="text">
         <string>VFrame</string>
        </property>
       </widget>
      </item>
      <item row="1" column="12">
       <widget class="QLabel" name="labelProfileVFrameValue"/>
      </item>
      <item row="0" column="13" rowspan="2">
       <widget class="Line" name="line">
        <property name="orientation">
         <enum>Qt::Orientation::Vertical</enum>
        </property>
       </widget>
      </item>
      <item row="0" column="14">
       <widget class="QLabel" name="labelProfileKeypresses">
        <property name="text">
         <string>Keypresses</string>
        </property>
       </widget>
      </item>
      <item row="0" column="15">
       <widget class="QLabel" name="labelProfileKeypressesValue"/>
      </item>
      <item row="1" column="14">
       <widget class="QLabel" name="labelProfileGame">
        <property name="text">
         <string>Game</string>
        </property>
       </widget>
      </item>
      <item row="1" column="15">
       <widget class="QLabel" name="labelProfileGameValue"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Search</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="labelStartDate">
        <property name="text">
         <string>Start Date</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="DateEdit" name="dateEditStartDate"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="labelEndDate">
        <property name="text">
         <string>End Date</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="DateEdit" name="dateEditEndDate"/>
      </item>
      <item row="2" column="0">
       <widget class="QPushButton" name="pushButtonOutputFile">
        <property name="text">
         <string>Output File</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="lineEditOutputFile">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="3" column="0" colspan="2">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QPushButton" name="pushButtonSearch">
          <property name="text">
           <string>Search</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButtonCancel">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Cancel</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QProgressBar" name="progressBar"/>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>DateEdit</class>
   <extends>QDateEdit</extends>
   <header>Form/Controls/DateEdit.hpp</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>comboBoxProfiles</tabstop>
  <tabstop>pushButtonProfileManager</tabstop>
  <tabstop>dateEditStartDate</tabstop>
  <tabstop>dateEditEndDate</tabstop>
  <tabstop>pushButtonOutputFile</tabstop>
  <tabstop>lineEditOutputFile</tabstop>
  <tabstop>pushButtonSearch</tabstop>
  <tabstop>pushButtonCancel</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
